server/container_create.go

```diff
@@ -651,8 +651,8 @@ func (s *Server) setupSeccomp(ctx context.Context, specgen *generate.Generator,
 		return nil
 	}
 
-	// Load the default seccomp profile from the server if the profile is a default one
-	if profile == seccompRuntimeDefault || profile == seccompDockerDefault {
+	// Load the default seccomp profile from the server if the profile is a default one or empty
+	if profile == "" || profile == seccompRuntimeDefault || profile == seccompDockerDefault {
```

Comment:
I would be good if we could add this behavior to criocli.go:
https://github.com/cri-o/cri-o/blob/5996f36f459ea2143b50dc98468e27ef69487e0f/internal/pkg/criocli/criocli.go#L463-L464

Reply:
> If not specified, then the internal default seccomp profile will be used

doesn't that imply the behavior?

Reply:
Somehow yes. 😬